vcf/filter.go

```diff
@@ -209,6 +209,32 @@ func IsPolarizable(v Vcf) bool {
 	return true
 }
 
+//IsWeakToStrong returns true if an input biallelic substitution variant is a weak to strong variant, false otherwise.
+func IsWeakToStrong(v Vcf) bool {
```

Comment:
I disagree with the name of this function. WeakToStrong implies Ref = A/T, Alt = C/G. What you are really testing is whether there is a A/T to C/G change regardless of ref/alt status. I think this needs to be either renamed into something more explicit (I think of `ChangesBondStrength` but that is admittedly not great) or make both a `WeakToStrong` and a `StrongToWeak`.

Reply:
Fixed.